shared/templates/rsyslog_logfiles_attributes_modify/ansible.template

```diff
@@ -0,0 +1,70 @@
+# platform = Red Hat Virtualization 4,multi_platform_fedora,multi_platform_ol,multi_platform_rhel,multi_platform_sle
+# reboot = false
+# strategy = configure
+# complexity = low
+# disruption = medium
+
+{{{ ansible_instantiate_variables(PARAMETER) }}}
+
+- name: "Set rsyslog logfile configuration facts"
+  set_fact:
```

Comment:
Please, use the complete name for the Ansible modules.
For example: `ansible.builtin.set_fact` instead of just `set_fact`.

Reply:
:+1: Should be handled in [b8a5ced](https://github.com/ComplianceAsCode/content/pull/9789/commits/b8a5ced30308275bb7f443e72ba91d3659b27946)